README.md

```diff
@@ -92,7 +95,10 @@ Check out the code in the [examples](./examples) folder: master,workers, tasks a
 * `stat = await w_exists(path, watch_cb)`
 * `data = await w_get(path, watch_cb)`
 * `children = await w_get_children(path, watch_cb)`
-* `{children, stat} = await w_get_children2 (path, watch_cb)`
+* `[children, stat] = await w_get_children2 (path, watch_cb)`
```

Comment:
I might be wrong, but the watch methods of Zookeeper Client do not return any values. They just call the callback provided to the method by the user.

Reply:
Ah, good catch! I'll fix that.

Reply:
I have verified that there are return values, and also that the callback is fired on changes.